sdk/tools/winesync/d3dx9_staging/0001-d3dx9__Return_D3DFMT_A8R8G8B8_in_D3DXGetImageInfoFromFileInMemory_for_32_bpp_BMP_with_alpha.diff

```diff
@@ -0,0 +1,73 @@
+diff --git a/dll/directx/wine/d3dx9_36/surface.c b/dll/directx/wine/d3dx9_36/surface.c
```

Comment:
These are just the staging diffs, right? If so, we probably don't want to check them into our repo?

Reply:
These are the staging diffs, well ordered and adapted to our tree and with the changes which are specific to our buils system or forked files (like the .spec ones you mention elsewhere)

The goal is to be able to easily revert them once you want to start a new update cycle, which need to be done on a "non-staged" base (to be implemented in the script). I considered maintaining a list of reactos commit, but then you have a chicken and egg problem, as you don't know the commit ID when you are committing

Reply:
You can see how they are used in the updated script.